core/src/main/java/org/apache/carbondata/core/metadata/schema/table/DataMapSchema.java

```diff
@@ -21,7 +21,14 @@
 import java.io.IOException;
 import java.io.Serializable;
 import java.util.HashMap;
+import java.util.HashSet;
```

Comment:
There is already a class name `DataMapSchema.java` so better change the older class name to `CarbonRowSchemq`